src/test/java/org/broadinstitute/hellbender/engine/FeatureDataSourceUnitTest.java

```diff
@@ -412,6 +411,16 @@ public void testQueryGVCF( final SimpleInterval queryInterval, final List<String
         }
     }
 
+    @Test(dataProvider = "IndependentFeatureQueryTestData")
+    public void testForNamedCodec (final SimpleInterval queryInterval, final List<String> UNUSED) {
```

Comment:
Add test case for `FeatureInput` that has a logical name/tag.

Reply:
Done!